### DISCUSSION TITLE
What is the default retry condition?

### DISCUSSION THREAD

**Discussion by fighterhit** [go-resty/resty#679] [Q&A] [answered]
I use the following  code, but it's not clear to me what is the default retry condition? Is it http status code >= 400 or is there an error in request execution?
```
httpClient = resty.New()
	httpClient.SetRetryCount(3).
		SetRetryWaitTime(time.Second).
		SetRetryMaxWaitTime(time.Second * 3)
```

**Comment by jeevatkm** [comment, marked as answer]
By default, Resty retry option values are - 
https://github.com/go-resty/resty/blob/1f11e18c7a061668ae1c3ae916090803ff03147d/retry.go#L17-L19

**Reply by fighterhit** [reply]
Thanks @jeevatkm , but what I mean is if I don't call `AddRetryCondition` function, will it retry by default? I guess not?
